Solink.AddIn.Helpers/RestartableAddInGenerator.cs

```diff
@@ -37,6 +37,7 @@ public RestartableAddInGenerator(string namespaceName, FileInfo sourceAssembly,
 
         public void Generate()
         {
+            AppDomain.CurrentDomain.ReflectionOnlyAssemblyResolve += (sender, args) => Assembly.ReflectionOnlyLoad(args.Name);

```

Comment:
Thanks, that should help make sure nobody accidentally deletes that line without breaking a test.